http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/resources/EncodeModeResource.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkus.ts.http.restclient.reactive.resources;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.reactive.client.api.ClientMultipartForm;
+import org.jboss.resteasy.reactive.server.multipart.MultipartFormDataInput;
+
+@Path("/encode")
+public class EncodeModeResource {
```

Comment:
I don't see anything related to mode in this test at all. If I am wrong, please give me a hint.

Reply:
I guess this is an outdated comment I'll resolve it, open again if I'm wrong.